src/BenchmarkDotNet/ConsoleArguments/CommandLineOptions.cs

```diff
@@ -175,9 +175,6 @@ public class CommandLineOptions
         [Option("wasmEngine", Required = false, HelpText = "Full path to a java script engine used to run the benchmarks, used by Wasm toolchain.")]
         public FileInfo WasmJavascriptEngine { get; set; }
 
-        [Option("wasmMainJS", Required = false, HelpText = "Path to the main.js file used by Wasm toolchain. Mandatory when using \"--runtimes wasm\"")]
-        public FileInfo WasmMainJs { get; set; }
```

Comment:
Please keep in mind that once we delete this parameter and you update BDN in dotnet/performance everything that is still using this argument is going to get an error from the BDN config parser

Reply:
Yes, I am aware. We have made changes in dotnet/performance so nothing is using it.